recipes/serpenttools/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "serpentTools" %}
+{% set version = "0.10.1" %}
+
+package:
+  # Do need the package name to be lower cased here, even though the
+  # package has that dang upper case T
+  name: "{{ name|lower }}"
```

```suggestion
  name: {{ name|lower }}
```
You can remove all of these `"` here and in the other parts of the recipe. Doesn't make a functional difference but just more uniform to read as we are doing the same in all other recipes.

---

Cleaned up with `grayskull`